### DISCUSSION TITLE
Watch command - add_collection

### DISCUSSION THREAD

**Discussion by dachewster99** [Taxel/PlexTraktSync#814] [Support] [answered]
Does the `watch` command also watch for library updates and add them to the collection? I know by default the answer to that is no, but I had edited the `config.json`, and I made the assumption that `add_collection` would also any newly added items it discovered the my `collection`. Though I am seeing now that it is not, and I was wondering if this is a bug, or if this is the intended functionality and I am just mistaken about its purpose

![image](https://user-images.githubusercontent.com/35871585/154668974-bcdb87af-1e30-458e-b6f8-e04f23347ba4.png)

**Comment by glensc** [comment, marked as answer]
it should work. not tested recently.
